similar_dev_search/cli/fetch_repos.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import click
+from joblib import Parallel, delayed
+
+from similar_dev_search.constants import JSONS_PATH, N_JOBS, REPOS_PATH
+from similar_dev_search.services.file_system import FileSystemService, JsonService
+from similar_dev_search.services.git import GitService, GithubService
+from similar_dev_search.services.git import RepositoryProvider
+from similar_dev_search.setup import setup
+
+
+@click.command()
+@click.option('--username', default='scikit-learn', help='Username for start repo')
+@click.option('--reponame', default='scikit-learn', help='Start repo name')
+@click.option('--max_repos', default=3, help='Max count of repos')
+def fetch_repos(username: str, reponame: str, max_repos: int) -> None:
+    print("Searching repositories...")
+    repos_list = list(find_repos(username, reponame))
+    print("Searching done...")
+    repositories_count = FileSystemService.get_file_count(JSONS_PATH)
+    Parallel(n_jobs=N_JOBS)(
+        delayed(handle_repo)(repo.owner.login, repo.name)
+        for repo in repos_list[:max(0, max_repos - repositories_count + 1)]
+    )
+
+
+def find_repos(username, reponame, depth=2):
+    ghs = GithubService()
+    repo = ghs.get_repo(username, reponame)
+    contributors = repo.get_contributors()
+    result = {repo}
+    if depth != 2:
```

Comment:
hardcode values - make them an arguments + docstring - it's not clear why you limit to 5 and so on